### PR TITLE
Extra logging fields

### PR DESCRIPTION
**Pull request by YonegoWillem**
An extra function that returns a GCEFormatter with added parameters to log extra fields like version number.

### CODE REVIEW

File: gceformatter.go [hunk 1]
```diff
@@ -87,12 +87,17 @@ func getSkipLevel(level logrus.Level) (int, error) {
 
 type GCEFormatter struct {
 	withSourceInfo bool
+    logData		   map[string]interface{}
```
Comment (pkartner) [inline gceformatter.go]: indenting seems off, maybe this file has tab indenting?

File: gceformatter.go [hunk 2]
```diff
@@ -110,6 +115,10 @@ func (f *GCEFormatter) Format(entry *logrus.Entry) ([]byte, error) {
 	data["severity"] = levelsLogrusToGCE[entry.Level]
 	data["logMessage"] = entry.Message
 
+	for key, value := range f.logData {
```
Comment (pkartner) [inline gceformatter.go]: doesn't this panic when you create a normal formatter?
Also indenting.

Comment (YonegoWillem) [reply gceformatter.go]: It won't panic. A nil map has a capacity of 0 so it wil snap out of the loop immediately if the map is nil.

File: gceformatter.go (hunk 1 above)
Comment (YonegoWillem) [reply gceformatter.go]: Jep tabs weren't changed to spaces...

**Pull request merged**